examples/langmuir_turbulence.jl

```diff
@@ -59,14 +59,13 @@ const vertical_scale = wavelength / 4π
 const Uˢ = amplitude^2 * wavenumber * frequency # m s⁻¹
 
 # The `const` declarations ensure that Stokes drift functions compile on the GPU.
-# To run this example on the GPU, write `architecture = GPU()` in the constructor
-# for `NonhydrostaticModel` below.
+# To run this example on the GPU, one only needs to include `architecture = GPU()` in the
+# constructor for `RectilinearGrid` above.
 #
-# The Stokes drift profile is
+# We only need the vertical derivative of the Stokes drift, which is
 
-uˢ(z) = Uˢ * exp(z / vertical_scale)
+∂z_uˢ(z, t) = 1 / vertical_scale * Uˢ * exp(z / vertical_scale)
 
-# which we'll need for the initial condition.
```

Comment:
Note that I changed a couple of things in this example. For example `uˢ(z)` wasn't used anywhere and didn't need to be defined. Also the message above incorrectly said that the `GPU()` option went into the `NonhydrostaticModel` constructor, which isn't the case anymore.

Reply:
We have to define the Stokes drift. We could do it with a comment rather than code, but code is better and no less clear. Can you put it back?